docs/docs/how-to/testing/visual-testing-with-storybook.md

```diff
@@ -131,29 +141,6 @@ window.___navigate = pathname => {
 
 > **Note:** Storybook's `preview.js` file allows you to set up decorators, parameters, and other aspects that affect the story rendering in the Canvas. Read more about it in the [official documentation](https://storybook.js.org/docs/react/configure/overview#configure-story-rendering).
 
-### Using an addon
```

Comment:
You'll need to remove the mentioning of this section, e.g. in the "Configuration" paragraph where this is linked